src/Core/Settings/GlobalSettings.cs

```diff
@@ -519,4 +522,74 @@ public class DistributedIpRateLimitingSettings
         public int SlidingWindowSeconds { get; set; } = 120;
     }
 
+    public class LogLevelSettings : ILogLevelSettings
+    {
+        public IBillingLogLevelSettings BillingSettings { get; set; } = new BillingLogLevelSettings();
```

Comment:
Maybe we should consider having project-level settings, given that these are not really "global" settings? No change required for this PR though.

Reply:
Yeah, @trmartin4 and I discussed this. We very likely will want to move in that direction. Benefit is that we don't have a bunch of settings that mean nothing, Cost is we can't just use the same settings file everywhere.